src/guided_fuzzing_daemon/nyx.py

```diff
@@ -63,7 +63,7 @@ def update_and_write(
         any_stat = False
 
         def convert_num(num: str) -> Union[float, int]:
```

Comment:
Why not always just return a float?

Reply:
This preserves the AFL output format (int stays int in the stats output) without hardcoding which is which.